src/components/ToolbarButton/ToolbarButton.module.scss

```diff
@@ -0,0 +1,72 @@
+.toolbarButton {
+  box-sizing: content-box;
+  vertical-align: top;
+  position: relative;
+  font-size: 1.2em;
+
+  width: 1.45em;
+  height: 1.25em;
+  margin-right: 0.4em;
+  padding: 0.2em 0.5em;
+
+  cursor: pointer;
+  border: 1px solid #ccc;
+  border-radius: 0.25em;
+  color: #333;
+  background: linear-gradient(to bottom, #fff 0, #f2f2f2 100%);
+
+  &:hover {
```

Comment:
```suggestion
  &:hover,
  &:focus {
```

Reply:
Did this so that a focused tooltip and a hovered tooltip will not overlap each other. But if you think this is better for accessibility we can add it